Yini-He/abstracts_bios_2019.md

```diff
@@ -1,6 +1,21 @@
 # Abstracts and bios
 
 This is a file of abstracts and applications for events I attend in 2019.
+### ABC2019
+
+**Website**: https://www.neuroscience.cam.ac.uk/events/ABC2019/pages/about-abc.html
+
+**Date**: 30 August 2019
```

Comment:
I think it makes more sense to be able to look back on this file knowing what date the _conference_ was, rather than the date that you added the information.

```suggestion
**Date**: 12 & 13 September 2019
```